src/components/matches/Scanner/Scanner.tsx

```diff
@@ -99,10 +104,10 @@ const Scanner = ({ forceUpdate }: { forceUpdate: () => void }) => {
     >
       <ScannerTutorial />
       <ScannerFeedback
-        open={feedbackVisible}
-        severity={feedbackSeverity}
-        feedback={feedback}
-        handleClose={() => setFeedbackVisible(false)}
+        open={feedback.visible}
+        severity={feedback.severity}
+        feedback={feedback.text}
+        handleClose={() => setFeedback({ ...feedback, visible: false })}
```

Comment:
```suggestion
        handleClose={() => setFeedback((previous) => ({ ...previous, visible: false }))}
```

Okay I'm sure it has no effect in this instance, but my eyes hurt whenever someone does a state update based on the current state at update-queue time, not at the actual update time 🙃

Reply:
Fixed in: https://github.com/boklisten/bl-next/pull/371/commits/f0a8bcf843c36ff791cb14d05f7da05c8bdb8ea1